agent/cmd/edgemesh-agent/app/config/config.go

```diff
@@ -38,6 +44,16 @@ type EdgeMeshAgentConfig struct {
 	Modules *Modules `json:"modules,omitempty"`
 }
 
+// CommonConfig defines some common configuration items
+type CommonConfig struct {
```

Comment:
Why do we need a commonConfig which just contain the config of dummy device? Can we just set the config in modules.edgedns?